### PR TITLE
make it run-able

### PR DESCRIPTION
**Pull request by fleeto**

### CODE REVIEW

File: artifacts/examples/example-tapp.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: apps.tkestack.io/v1
+apiVersion: tke.cloud.tencent.com/v1
```
Comment (hex108) [inline artifacts/examples/example-tapp.yaml:1]: Why do we need change the group name?

File: deployment/tapp-controller.yaml [hunk 2]
```diff
@@ -43,7 +43,26 @@ spec:
...
             - "--register-admission=true"
           imagePullPolicy: "Always"
+          ports:
+          - containerPort: 8443
+            protocol: "TCP"
+            name: "https"
           resources:
             requests:
-              "cpu": "1"
+              "cpu": "100m"
```
Comment (hex108) [inline deployment/tapp-controller.yaml:52]: 100m is too small

**Review by hex108 — COMMENTED** [review]
Thanks!

File: artifacts/examples/example-tapp.yaml (hunk 1 above)
Comment (fleeto) [reply artifacts/examples/example-tapp.yaml:1]: I'd deployed tapp controller via [tapp-controller.yaml](https://github.com/tkestack/tapp/blob/master/deployment/tapp-controller.yaml)

image

File: deployment/tapp-controller.yaml (hunk 2 above)
Comment (fleeto) [reply deployment/tapp-controller.yaml:52]: So it can startup on my tiny experiment cluster

File: artifacts/examples/example-tapp.yaml (hunk 1 above)
Comment (hex108) [reply artifacts/examples/example-tapp.yaml:1]: Sorry, the CRD in tapp-controller.yaml is wrong. It will be better change it.

**Pull request closed without merging**